src/tests/ftest/pool/evict_test.py

```diff
@@ -86,19 +86,10 @@ def evict_badparam(self, test_param):
         self.pool = self.connected_pool(self.hostlist_servers)
 
         self.log.info(
-            "Pool UUID: %s\n Pool handle: %s\n Server group: %s\n",
-            self.pool.uuid, self.pool.pool.handle.value, self.pool.name)
+            "Pool UUID: %s\n Pool handle: %s\n",
+            self.pool.uuid, self.pool.pool.handle.value,)
```

Comment:
```suggestion
            self.pool.uuid, self.pool.pool.handle.value)
```

Reply:
thanks will fix this in a follow-up as it doesn't cause any functional problems

Reply:
fix to be applied in https://github.com/daos-stack/daos/pull/5436